examples/gotext/asa.yml

```diff
@@ -0,0 +1,187 @@
+  - "id": "305011"
+    "value": "{{if .IncludeTimestamp}}{{TimestampFormatter \"Jan 02 2006 03:04:05.123\" \"10s\"}}: {{end}}%ASA-6-305011: Built {{.TranslationType}} {{.Protocol}} translation from {{.SrcInt}}:{{.SrcAddr}}/{{.SrcPort}} to {{.DstInt}}:{{.DstAddr}}/{{.DstPort}}"
+
+fields:
+  - name: AccessGroup
+    type: string
+    choices:
+      - "Access-Group"
+  - name: AclId
+    type: string
+    choices:
+      - "AclId"
+  - name: Bytes
+    type: int
+  - name: Code
+    type: int
+  - name: ConnectionId
+    type: int
+  - name: Direction
+    type: string
+    choices:
+      - "inbound"
+      - "outbound"
+  - name: DstAddr
+    type: ipv4
+  - name: DstInt
+    type: interface
+  - name: DstPort
+    type: port
+  - name: DstUser
+    type: string
+    choices:
+      - "DstUser"
+  - name: Duration
+    type: duration
+  - name: IncludeTimestamp
+    type: bool
+  - name: Map1Addr
+    type: ipv4
+  - name: Map1Port
+    type: port
+  - name: Map2Addr
+    type: ipv4
+  - name: Map2Port
+    type: port
+  - name: Protocol
+    type: string
+    choices:
+      - "TCP"
+      - "UDP"
+  - name: Reason
+    type: string
+    choices:
+      - "Conn-timeout"
+      - "Deny Terminate"
+      - "Failover primary closed"
+      - "FIN Timeout"
+      - "Flow closed by inspection"
+      - "Flow terminated by IPS"
+      - "Flow reset by IPS"
+      - "Flow terminated by TCP Intercept"
+      - "Flow timed out"
+      - "Flow timed out with reset"
+      - "Flow is a loopback"
+      - "Free the flow created as result of packet injection"
+      - "Invalid SYN"
+      - "IPS fail-close"
+      - "No interfaces associated with zone"
+      - "No valid adjacency"
+      - "Pinhole Timeout"
+      - "Probe maximum retries of retransmission exceeded"
+      - "Probe maximum retransmission time elapsed"
+      - "Probe received RST"
+      - "Probe received FIN"
+      - "Probe completed"
+      - "Route change"
+      - "SYN Control"
+      - "SYN Timeout"
+      - "TCP bad retransmission"
+      - "TCP FINs"
+      - "TCP Invalid SYN"
+      - "TCP Reset - APPLIANCE"
+      - "TCP Reset - I"
+      - "TCP Reset - O"
+      - "TCP segment partial overlap"
+      - "TCP unexpected window size variation"
+      - "Tunnel has been torn down"
+      - "Unauth Deny"
+      - "Unknown"
+      - "Xlate Clear"
+  - name: RewindValue
+    type: string
+    choices:
+      - "10s"
+      - "2h30m"
+  - name: SrcAddr
+    type: ipv4
+  - name: SrcInt
+    type: interface
+  - name: SrcPort
+    type: port
+  - name: SrcUser
+    type: string
+    choices:
+      - "SrcUser"
+  - name: Timestamp
+    type: time.Time
+  - name: TranslationType
+    type: string
+    choices:
+      - "dynamic"
+      - "static"
+  - name: Type
+    type: int
+
+# // New is Factory for the asa generator
```

Comment:
I'm guessing we don't need these comments.  Right?

Reply:
true